src/modules/Common/managers/ServiceManager.ts

```diff
@@ -146,7 +161,9 @@ You can load it [on your local instance](${localUrl}) if you have one set up._
         targetBranch: 'main',
         newBranch: branchName,
         title: prTitle,
+        message: prTitle,
```

Comment:
Is this really expected? It seems surprising that the `message` would be the same as the `title`.

Reply:
As message is "Add <service> <documentType>" I believe it is explanatory enough

In order to be coherent with other commits, I would have to use "Add <service> <documentType> declaration" which I can do.